_posts/2019-03-05-build-firefox-extension-with-ringcentral-embeddable.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: post
+title:  "Building a Firefox add-on with RingCentral Embeddable"
+date:   2019-03-05 10:00:00 +0800
+categories: blog
+author: Embbnux Ji
+---
+
+![ringcentral-embeddable-with-firefox](https://user-images.githubusercontent.com/7036536/55455660-0e8f0000-5617-11e9-89be-17c008c7d3f4.png)
+
+In our previous [article](https://medium.com/ringcentral-developers/build-a-chrome-extension-with-ringcentral-embeddable-bb6faee808a3), we introduce how to use RingCentral Embeddable to build a Chrome extension to integration RingCentral voices and text features into any CRM website. Now RingCentral Embeddable has also supported Firefox. So now we can also use Firefox add-on to integrate RingCentral into any CRM website.
```

Comment:
we `introduced` how to use ... to build a Chrome extension to `integrate` RingCentral voices and text features into any CRM `websites`. Now RingCentral Embeddable `also has support for Firefox, so` we can also use Firefox ... into any CRM `websites`.